script.js

```diff
@@ -310,7 +334,8 @@ function handleData([
   moratoriumSheetsText,
```

Comment:
@jimboweb with the above cities sql query you should be able to remove the `fetch()` call to the now deprecated moratoriums sheet.